vertx-mysql-client/src/main/asciidoc/index.adoc

```diff
@@ -99,6 +99,13 @@ A simple way to configure the client is to specify a `MySQLConnectOptions` data
 {@link examples.MySQLClientExamples#configureFromDataObject(io.vertx.core.Vertx)}
 ----
 
+You can also configure the connection attributes with the `setProperties` or `addProperty` methods. Note `setProperties` will override the default client properties.
```

Comment:
can you add a link to MySQL connection attributes documentation to make this clearer ?

Reply:
fixed in https://github.com/eclipse-vertx/vertx-sql-client/pull/344/commits/ab1bc0677847338f11ee7d3ed2d87fbd7812b296.